### DISCUSSION TITLE
[General] join discord <Title>

### DISCUSSION THREAD

**Discussion by Ali3447** [zkSync-Community-Hub/zksync-developers#303] [General] [answered]
### Environment

Mainnet

### Provide a brief description of the functionality you're trying to implement.

why we can not join to discord zksync ?
![zksynce](https://github.com/zkSync-Community-Hub/zksync-developers/assets/100008404/7c879732-7bd1-4c23-8646-6973c31185b5)

### What is the specific issue or error you're encountering?

why 

### Can you share the error messages or logs you're receiving, if any?

we can not 

### Have you made any recent changes to the contract before encountering this issue?

join 

### Are there any external libraries or contracts that your contract interacts with?

rttttto your 

### Can you provide the relevant portions of your contract code where the issue is occurring?

discord 

### Have you tried to isolate the problem, and if so, what were the results?

why 

### What steps have you already taken to try to resolve the issue?

why

**Comment by bxpana** [comment, marked as answer]
Please email us at support@zksync.io and we can help you there. This is a place for Developer support

**Reply by Ali3447** [reply]
> Please email us at support@zksync.io and we can help you there. This is a place for Developer support

Tanks my friend 🥰
